tests/x11/remote_desktop/windows_client_remotelogin.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Remote Login: Windows access SLES over RDP
+# Maintainer: GraceWang <gwang@suse.com>
+# Tags: tc#1610388
+
+use strict;
+use base 'x11test';
+use testapi;
+use lockapi;
+
+sub run {
+    my $self = shift;
+
+    mutex_lock 'xrdp_server_ready';
+
+    send_key "super-r";
+    assert_screen "windows-run";
+    type_string "mstsc\n";
+    assert_screen "remote-desktop-connection";
+    type_string '10.0.2.17';
+    wait_still_screen 3;
+    send_key 'ret';
+    sleep 60;
```

Comment:
I don't think we would need something like this. Why not just the following assert_screen?

Reply:
Updated to "assert_screen".
But new verification run and needle were not provided because of the "Could not resolve host: openqa.suse.de" error in http://10.67.18.241/tests/1111#step/xrdp_server/31